brightwind/utils/utils.py

```diff
@@ -176,4 +183,67 @@ def is_file(file_or_folder):
         raise FileNotFoundError("File or folder doesn't seem to exist.")
 
 
+def is_extension(file_or_folder, extension_required):
+    """
+    Returns True if file_or_folder is a file of the desired extension type.
+
+    :param file_or_folder:      The file or folder path.
+    :type file_or_folder:       str
+    :param extension_required:  The file extension needed.
+    :type extension_required:   str
+    :return:                    If is a file with desired extension type.
+    :rtype:                     bool
+    """
+    if is_file(file_or_folder):    
+        _, extension = os.path.splitext(file_or_folder)
+        if extension.lower() == extension_required:
+            return True
+        else:
+            raise ValueError(f"File extension must be {extension_required}, got: {extension}")
+    else:
+        raise ValueError(f"Input must be a {extension_required} file, got: {file_or_folder}")
+
+
 
+def check_schema(json_to_check, schema):
```

Comment:
can you add tests for this function under test_utils.py?

Reply:
actually no need for this as I see that you have added tests for this in test_load.py when checking cleaning_rules schema

Reply:
actually no need for this as I see that you have added tests for this in test_load.py when checking cleaning_rules schema